rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```diff
@@ -21,6 +21,11 @@ find_package(PythonInterp 3.6 REQUIRED)
 
 find_package(python_cmake_module REQUIRED)
 find_package(PythonExtra MODULE REQUIRED)
+set(_PYTHON_EXECUTABLE ${PYTHON_EXECUTABLE})
+if(WIN32 AND "${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
+  set(PYTHON_EXECUTABLE ${PYTHON_EXECUTABLE_DEBUG})
```

Comment:
I'd guess setting `Python3_EXECUTABLE` to `PYTHON_EXECUTABLE_DEBUG` before `find_package(Python3 ...)` would allow using the the debug version of NumPY on Windows

https://cmake.org/cmake/help/latest/module/FindPython3.html#artifacts-specification

Reply:
Similar to here https://github.com/ros2/rclpy/blob/7ffe94865f38c67812799bf5751a8f0f3f918c25/rclpy/CMakeLists.txt#L17-L24

Reply:
That's exactly what I wanted to do, but I didn't do it correctly.